packages/rum-core/src/domain/assembly.spec.ts

```diff
@@ -298,7 +298,7 @@ describe('rum assembly', () => {
         rawRumEvent: createRawRumEvent(RumEventType.ACTION),
         startTime: 0 as RelativeTime,
       })
-      expect((serverRumEvents[0] as RumActionEvent).action.id).not.toBeDefined()
+      expect((serverRumEvents[0] as RumActionEvent).action.id).toBeDefined()
```

Comment:
what is the intent of this expectation?

Reply:
Shouldn't the id be required for all actions?

Reply:
yes, but it is not the goal of this test.
In the context of this test, the idea is to check by event type is the action context is added to the event or not, meaning testing [this function](https://github.com/DataDog/browser-sdk/blob/abccf2c/packages/rum-core/src/domain/assembly.ts#L127)

Reply:
okay done ;)